src/app/utils/date.ts

```diff
@@ -19,3 +19,23 @@ export const createQueryWithDateParam = (
     return {}
   }
 }
+
```

Comment:
nitpick, but maybe this should be in the examples module since the formatting is specific to it

Reply:
I'd have thought the formatting is relatively generic, but yea since it's only used there i'll move it